react-frontend/src/pages/ActiveListingsPage.js

```diff
@@ -22,7 +23,19 @@ const ActiveListingsPage = () => {
       }
     }
 
+    const fetchUserWatchlist = async () => {
+      try {
+        const data = await fetchWatchlistIds()
+        setWatchlist(data)
+      } catch (error) {
+        console.error('Error fetching watchlist IDs:', error)
+      }
+    }
+
     fetchListings().catch((error) => console.error('Unhandled error in fetchListings:', error))
+    fetchUserWatchlist().catch((error) =>
+      console.error('Unhandled error in fetchUserWatchlist:', error)
```

Comment:
Let's add `error state` to show the error message in case of any troubles

Reply:
Added error state in [b655d78](https://github.com/YaroslavKSE/AuctionWeb/pull/3/commits/b655d78aa4cf807c011c5b5bb5f81bf9be9e0e84)